react-native-libraries.json

```
@@ -14468,5 +14468,12 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/xxsnakerxx/react-native-alert-queue",
+    "npmPkg": "react-native-alert-queue",
+    "ios": true,
+    "android": true,
+    "expoGo": true,
```

```suggestion
    "githubUrl": "https://github.com/xxsnakerxx/react-native-alert-queue",
    "ios": true,
    "android": true,
    "expoGo": true
```

Syntax fix + entry cleanup.